src/hotspot/share/gc/serial/defNewGeneration.inline.hpp

```diff
@@ -91,11 +91,12 @@ inline void DefNewGeneration::FastKeepAliveClosure::do_oop_work(T* p) {
 
 template <typename OopClosureType>
 void DefNewGeneration::oop_since_save_marks_iterate(OopClosureType* cl) {
-  eden()->oop_since_save_marks_iterate(cl);
-  to()->oop_since_save_marks_iterate(cl);
-  from()->oop_since_save_marks_iterate(cl);
+  // no allocation in from-space (eden/from)
```

Comment:
I found the comment a bit confusing (and not conforming to style), here's an attempt to improve.
```suggestion
  // No allocation in eden and from-space, so no iteration required.
```

Reply:
I was using the `from-space` term from copying GC algo to refer both eden and from spaces in the code.